normandy/recipes/filters.py

```diff
@@ -353,6 +353,28 @@ def capabilities(self):
         return {"jexl.context.env.version", "jexl.transform.versionCompare"}
 
 
+class DateRangeFilter(BaseFilter):
```

Comment:
Sorry, I forgot this in the previous review. Can you add a doc string to this class, like the others have? It is used to automatically generated documentation that gets published here: https://mozilla.github.io/normandy/user/filters.html#module-normandy.recipes.filters

Reply:
Ok I added one, not sure the language is the best it could be... Can you double check?

Reply:
This looks good to me. I'm a little unsure about the syntax of the RST in the comment, but that's fine for now. It turns out that this isn't being automatically included in the docs anyways. I'm going to look into that in another PR (unless you beat me to it).